src/ui/menu.ts

```diff
@@ -117,6 +118,29 @@ async function menu(_ui: UI) {
               return;
             });
         return;
+      },
+      popOut: () => {
+        let windowType;
+        if (navigator.userAgent.indexOf('Firefox') !== -1) {
+          windowType = 'detached_panel';
+        } else {
+          windowType = 'panel';
+        }
+        chrome.windows.create({
+          url: chrome.extension.getURL('view/popup.html'),
+          type: windowType,
+          height: 480,
+          width: 320
```

Comment:
It seems here's a bug of Chrome, the opened window is not in width of 320px

Reply:
It's not a big problem, we can fix it in the later version.

Reply:
Looks like if the size is updated after creating the window it might fix the issue?

I also found an old bug, but it's probably irrelevant. https://bugs.chromium.org/p/chromium/issues/detail?id=173831